src/Handler.php

```diff
@@ -12,6 +12,20 @@ class Handler
     protected $client;
 
     /**
+     * The previously registered error handler returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousErrorHandler;
+
+    /**
+     * The previously registered exception handler returned by the interpreter.
```

Comment:
I think `The previously registered exception handler.` is probably fine, and similar for above. While the main implementation of the PHP language is an interpreter (after a compilation phase to an intermediate language), there exist other implementations of the language, so no reason to assume the runtime is actually an interpreter. Just something very minor. :P